hipify_torch/cuda_to_hip_mappings.py

```diff
@@ -8938,7 +8938,9 @@
         # This substitution is not permissible, because there's another copy of this
         # function in torch/cuda.h
         # ("cuda::device_count", ("hip::device_count", API_C10)),
-        ("cuda::current_device", ("hip::current_device", API_C10)),
+        # This one should go with the previous one for Transformer Engine
```

Comment:
This breaks torch hipify 
https://github.com/ROCmSoftwarePlatform/pytorch/blob/rocm6.0_internal_testing/aten/src/ATen/test/cuda_stream_test.cpp#L127